cmd/sync_chart.go

```diff
@@ -48,7 +48,7 @@ func newSyncChart() *cobra.Command {
 
 func syncChart() error {
 	if !syncAllVersions && version == "" {
-		return errors.Trace(fmt.Errorf("Please specify a version to sync with --version VERSION or sync all versions with --all-versions"))
+		return errors.Trace(fmt.Errorf("please specify a version to sync with --version VERSION or sync all versions with --all-versions"))
 	}
```

Comment:
The right way is adding a `PreRunE` hook to the cmd. Example:

```
cmd := &cobra.Command{
    ...
    PreRunE: func(cmd *cobra.Command, args []string) error {
        if !syncAllVersions && version == "" {
            return errors.Trace(fmt.Errorf(`missing "--version" flag: Please use either "--version VERSION" or "--all-versions"))
        }
        if syncAllVersions && version != "" {
            klog.Warningf(`Ignoring "--version" flag: "--all-versions" is set))
        }
        return nil
    },
```

Reply:
done!